clifford/cga.py

```diff
@@ -1,6 +1,6 @@
-
 '''
 .. currentmodule:: clifford.cga
+
```

Comment:
Chanegs in this file only to make sphinx not emit warnings - rendering is the same